src/main/java/core/basesyntax/dao/writeReport.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax.dao;
+
+public interface writeReport {
```

Comment:
The interface name `writeReport` does not follow Java naming conventions. It should be renamed to `WriteReport` to start with an uppercase letter.